angular-2-components/src/app/features/registration/registration.component.html

```diff
@@ -0,0 +1,85 @@
+<div>
+  <app-header> </app-header>
+  <div class="registration_container">
+    <form [formGroup]="formDataRegistration" (ngSubmit)="onSubmit()">
+      <legend class="title">Registration</legend>
+      <fieldset class="registration_fieldset">
+        <label class="label" for="account_nickname">
+          Name
+          <input
+            id="account_nickname"
+            class="input"
+            type="text"
+            required
+            formControlName="account_nickname" />
+          <div
+            class="alert"
+            *ngIf="
+              account_nickname.errors &&
+              (account_nickname.dirty || account_nickname.touched)
+            ">
+            <p *ngIf="account_nickname.errors?.['required']">
+              Name is required
+            </p>
+          </div>
+        </label>
+        <label class="label" for="account_email">
+          Email
+          <input
+            id="account_email"
+            class="input"
+            type="email"
+            required
+            formControlName="account_email"
+            appEmailValidate />
+          <div
+            class="alert"
+            *ngIf="
+              account_email.errors &&
+              (account_email.dirty || account_email.touched)
+            ">
+            <p *ngIf="account_email.errors?.['required']">Email is required</p>
```

Comment:
Try to use shared string `enum `for all of this string such as `Email is required` and etcetera.